### PR TITLE
W-15585387 Add troubleshooting Azure guide.

### PR DESCRIPTION
**Pull request by Cristian-Venticinque**

### CODE REVIEW

File: rpa-builder/6.6/modules/ROOT/pages/troubleshooting-azure-oauth-setup.adoc [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+. <<verify-settings>>
+. <<add-permissions>>
+
+[[register-app]]
+== Register an App in Azure
+
+. Navigate to https://portal.azure.com/ and log in with your Azure account. 
+. Open the *App Registrations* page and create a new registration. 
+** Select the supported account types. 
+** Provide a Redirect URI.
```
Comment (ms-tlu) [inline rpa-builder/6.6/modules/ROOT/pages/troubleshooting-azure-oauth-setup.adoc:16]: This is a regular point of confusion for customers, I think we should emphasize that the URL needs to be any valid URL and should provide an example like http://localhost:49453/

**Review by ms-tlu — DISMISSED** [review]

File: rpa-builder/6.6/modules/ROOT/pages/troubleshooting-azure-oauth-setup.adoc [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+[[create-secret]]
+== Create a Client Secret for the App
+
+After registering an app in Azure, the app's setting page opens.
+
+. Open the *Certificates & Secrets* page.
+. Create a new client secret.
++
+[WARNING]
+After creating the secret, ensure you copy its *Value* as this is the only time you can see it. After you leave this page, the secret is not accessible anymore.
```
Comment (Crispy-Salesforce) [inline rpa-builder/6.6/modules/ROOT/pages/troubleshooting-azure-oauth-setup.adoc]:
```suggestion
After creating the secret, ensure you copy its *Value* as this is the only time you can see it. After you leave this page, the secret is no longer accessible. 
```

File: rpa-builder/6.6/modules/ROOT/pages/troubleshooting-azure-oauth-setup.adoc [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+After registering your app and creating its client secret, configure permission settings:
+
+. Open *API Permissions* in the side bar. 
+. Click *Add a Permission* and select *Microsoft Graph*. 
+. Select *Delegated Permissions* and add the following permissions from the *OpenId permissions* category: 
+** `email`
+** `offline_access`
+** `openid`
+** `profile`
+. Select the appropriate permissions and protocols depending on your Workflow needs:
```
Comment (Crispy-Salesforce) [inline rpa-builder/6.6/modules/ROOT/pages/troubleshooting-azure-oauth-setup.adoc]:
```suggestion
. Select the appropriate permissions and protocols depending on your workflow needs: 
```

Comment (Crispy-Salesforce) [reply rpa-builder/6.6/modules/ROOT/pages/troubleshooting-azure-oauth-setup.adoc]: If "Workflow" is the name of a feature, dismiss this suggestion.

File: rpa-builder/6.6/modules/ROOT/pages/troubleshooting-azure-oauth-setup.adoc [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
++
+--
+** Category `Mail`
+** Category `IMAP`
+** Category `EWS`
+** Category `POP`
+** Category `SMTP`
+--
++
+If you are not sure which of these apply to your case, select all of them.
```
Comment (Crispy-Salesforce) [inline rpa-builder/6.6/modules/ROOT/pages/troubleshooting-azure-oauth-setup.adoc]:
```suggestion
If you don't know the permissions and protocols that apply to your case, select all of them. 
```

**Review by Crispy-Salesforce — DISMISSED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**